PYME/LMVis/renderers.py

```diff
@@ -328,7 +328,7 @@ class Histogram3DRenderer(HistogramRenderer):
 
     def genIm(self, settings, imb, mdh):
         mdh['Origin.z'] = settings['zBounds'][0]
-        return visHelpers.rendHist3D(self.colourFilter['x'],self.colourFilter['y'], self.colourFilter['z'], imb, settings['pixelSize'], settings['zBounds'], settings['zSliceThickness'])
```

Comment:
Why this change???? We should respect the z bounds that we set in the  generate3D dialog

Reply:
Because `visHelpers.rendHist3D` doesn't accept a zbound argument. It has a max of 6 arguments.

Reply:
oops - that gets handled in `generate()`, although `generate()` sets `'Origin.z'` incorrectly.